ChangeLog

```diff
@@ -1,3 +1,17 @@
+Release 1.2rc1
+
+- New co-maintainer Michael Osipov
+- Improvement: allow to delete and reset computer account
+- Improvement: prefer SASL mechanism GSS-SPNEGO over GSSAPI for LDAP
+               connections to domain controllers (thanks, James Ralston!)
```

Comment:
I still don't understand the benefit of the change. In fact, I have commented on the commit, but the author did not respond to it.

Reply:
I've added a comment there. Hope this clarifies the motivation of this change.